src/clients/inst_autosetup.rb

```diff
@@ -337,8 +337,18 @@ def main
       return :abort if Popup.ConfirmAbort(:painless) if UI.PollInput == :abort
 
       Progress.NextStage
+
+      # Evaluating package and patterns selection.
+      # Selection will stored in PackageAI.
       AutoinstSoftware.Import(Ops.get_map(Profile.current, "software", {}))
```

Comment:
AutoinstSoftware.Import(Profile.current.fetch("software", {})) but you probably want to change as little as possible :)